web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
I think everyone can agree that the current page isn't optimal :) I think to get this "right", it'd be great to get a UI expert, but we can try to improve a bit on our own too. Still :+1: on this general change though for now.